Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM cibuilds/hugo:0.80 AS builder
+
+COPY content content
+COPY static static
+COPY themes themes
+COPY resources resources
+COPY config.toml .
+
+RUN hugo --minify
+
+FROM nginx:stable-alpine
+COPY --from=builder /home/circleci/project/public /usr/share/nginx/html
```

Comment:
just a noob question since I haven't done this before: are we supposed to set up a circleCI job on here then too?

Reply:
nope, I'm using image that CircleCI created. That's why the user is called like that.

Reply:
Ahhh ok!